### PR TITLE
refactor: move allowed types out of the parenthesis to make it more official

### PR DESCRIPTION
**Pull request by danielpeintner**
fixes https://github.com/w3c/wot-scripting-api/issues/403

***
<a href="https://pr-preview.s3.amazonaws.com/danielpeintner/wot-scripting-api/pull/414.html" title="Last updated on Jul 18, 2022, 12:23 PM UTC (2f2d1ca)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-scripting-api/414/cf3d575...danielpeintner:2f2d1ca.html" title="Last updated on Jul 18, 2022, 12:23 PM UTC (2f2d1ca)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -749,8 +749,8 @@ <h2>The <dfn>InteractionInput</dfn> type</h2>
     </p>
     <p>
       <dfn>DataSchemaValue</dfn> is an
-      <a href="https://www.ecma-international.org/ecma-262/11.0/index.html#sec-ecmascript-data-types-and-values">ECMAScript value</a> that is accepted for <a>DataSchema</a> defined in [[WoT-TD]]
-      (i.e. null, boolean, number, string, array, or object).
+      <a href="https://www.ecma-international.org/ecma-262/11.0/index.html#sec-ecmascript-data-types-and-values">ECMAScript value</a> that is accepted for <a>DataSchema</a> defined in [[WoT-TD]].
+	  The possible values are of type <code>null</code>, <code>boolean</code>, <code>number</code>, <code>string</code>, <code>array</code>, or <code>object</code>.
```
Comment (relu91) [inline index.html:753]:
```suggestion
	  The possible values are of type <a href="https://262.ecma-international.org/11.0/#sec-ecmascript-language-types-null-type">null</a>, <a href="https://262.ecma-international.org/11.0/#sec-ecmascript-language-types-boolean-type">boolean</a>, <a href="https://262.ecma-international.org/11.0/#sec-ecmascript-language-types-number-type">number</a>, <a href="https://262.ecma-international.org/11.0/#sec-ecmascript-language-types-string-type">string</a>, <a href="https://262.ecma-international.org/11.0/#sec-array-objects">array</a>, or <a href="https://262.ecma-international.org/11.0/#sec-object-type">object</a>.
```

**Review by relu91 — CHANGES_REQUESTED** [review]
I would use the ref to the definition of the types as shown in the comment below. Moreover, I am ok for using MUST.

File: index.html (hunk 1 above)
Comment (danielpeintner) [reply index.html:753]: 👍

**Review by relu91 — APPROVED** [review]

**Comment by danielpeintner** [comment]
Call 2022-07-18
* agreed to use MUST
* merge afterwards

**Pull request merged**